.github/workflows/pre-integration.yml

```diff
@@ -76,20 +76,21 @@ jobs:
         shell: pwsh
         run: Start-Sleep -s 30
 
-      - name: Launch integration tests 9.1
+      - name: Launch integration tests 10.4
         shell: pwsh
         run: |
           .github/submit/submit_test_job.ps1  `
-            -sparkVersion "9.1.x-scala2.12" `
-            -testJobDetails job91.json
+            -sparkVersion "10.4.x-scala2.12" `
+            -testJobDetails job104.json `
+            -sparkLibs sparklibs104.json
 
       - name: Wait 2 min for things to settle
         shell: pwsh
         run: Start-Sleep -s 120
 
-      - name: Fetch integration tests 9.1
+      - name: Fetch integration tests 10.4
```

Comment:
is there a need to fetch integrationtests from 9.1 too?

Reply:
indeed there is

Reply:
I agree.